ci/prow-build-test-qemu.sh

```diff
@@ -30,9 +30,11 @@ fi
 ocpver=$(rpm-ostree compose tree --print-only src/config/manifest.yaml | jq -r '.["mutate-os-release"]')
 ocpver_mut=$(rpm-ostree compose tree --print-only src/config/manifest.yaml | jq -r '.["mutate-os-release"]' | sed 's|\.|-|')
 prev_build_url=${REDIRECTOR_URL}/rhcos-${ocpver}/
-# we want to use RHEL 8.5 for testing until we can start using 8.6
-# see https://github.com/openshift/release/pull/26193
+# temporarily also fetch 8.5 repo for sssd
+# https://bugzilla.redhat.com/show_bug.cgi?id=2072050
 curl -L http://base-"${ocpver_mut}"-rhel86.ocp.svc.cluster.local > src/config/ocp.repo
+curl -L http://base-"${ocpver_mut}"-rhel86.ocp.svc.cluster.local > src/config/ocp85.repo
```

Comment:
This will probably work, but won't actually run the test with the 8.5 sssd.
Hmm, not sure why the previous iteration didn't work when it worked not long ago: https://github.com/openshift/os/commit/d53404b1ea427be814cb15d1f5b242b1c8de85a1.

Ahh, it's possible the `sed` is being too greedy. Based on:

> Updating rpm-md repo 'rhel-85-baseos': cannot update repo 'rhel-85-baseos': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried; Last error: Status code: 404 for http://base-4-11-rhel85.ocp.svc.cluster.local/rhel-85-baseos/repodata/repomd.xml (IP: 172.30.113.108)

it looks like the mirroring repo baseurl includes `rhel-8-` in it.

This might work instead:

```
sed -i -e 's,\[rhel-8-,\[rhel-85-,' src/config/ocp85.repo
```